dev/create-release/release-build.sh

```diff
@@ -382,9 +382,9 @@ if [[ "$1" == "package" ]]; then
     mkdir -p "svn-spark/${DEST_DIR_NAME}-bin"
 
     echo "Copying release tarballs"
-    cp spark-* "svn-spark/${DEST_DIR_NAME}-bin/"
-    cp pyspark-* "svn-spark/${DEST_DIR_NAME}-bin/"
-    cp SparkR_* "svn-spark/${DEST_DIR_NAME}-bin/"
+    cp spark* "svn-spark/${DEST_DIR_NAME}-bin/"
+    cp pyspark* "svn-spark/${DEST_DIR_NAME}-bin/"
```

Comment:
note: only pyspark tarballs have the issue, but I applied the fix to all of them so that it's more reliable.